### PR TITLE
⚡️ add pexpect lab

### PR DESCRIPTION
**Pull request by tigerza117**
Add lab pexpect

- [x] R1
    - [x] Connection
    - [x] Command send
    - [x] Setting LoopBack
    - [x] Show result
- [x] R2
    - [x] Connection
    - [x] Command send
    - [x] Setting LoopBack
    - [x] Show result
- [x] R3
    - [x] Connection
    - [x] Command send
    - [x] Setting LoopBack
    - [x] Show result

### CODE REVIEW

**Comment by tigerza117** [comment]
I approve my code, What do you think about this pr ?

File: pexpectlab.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import ipaddress
+
+import pexpect
+
+PROMPT = '#'
+IP = '10.0.15.111'
+USERNAME = 'admin'
+PASSWORD = 'cisco'
```
Comment (tigerza117) [inline pexpectlab.py:8]: Not good idea for leave password in open source project

**Review by SoNicBoom24 — COMMENTED** [review]
I think it's a good job

**Comment by tigerza117** [comment]
Approved!

**Review by SoNicBoom24 — APPROVED** [review]
Approve

**Pull request merged**